src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeRequest.java

```diff
@@ -76,6 +76,11 @@ public String text() {
         return this.text;
     }
 
+    public AnalyzeRequest text(String text) {
```

Comment:
I can't comment on the constructor, but can we have one without the text parameter? it's always null now . I'm okay with leaving the current constructor there as well...